sunpy/timeseries/sources/lyra.py

```diff
@@ -54,46 +54,68 @@ class LYRATimeSeries(GenericTimeSeries):
     # Class attribute used to specify the source class of the TimeSeries.
     _source = 'lyra'
 
-    @peek_show
-    def peek(self, names=3, **kwargs):
+    def plot(self, title=None, names=3, **kwargs):
         """
-        Plots the LYRA data. An example is shown below:
-
-        .. plot::
-
-            import sunpy.timeseries
-            import sunpy.data.sample
-            lyra = sunpy.timeseries.TimeSeries(sunpy.data.sample.LYRA_LEVEL3_TIMESERIES, source='LYRA')
-            lyra.peek()
```

Comment:
Restore this?

Reply:
It's part of peek in the method below, this is plot()